src/linker/Linker/Driver.cs

```diff
@@ -674,6 +673,16 @@ public bool Run (ILogger customLogger = null)
 
 			try {
 				p.Process (context);
+			} catch (Exception ex) {
+				if (ex is LinkerFatalErrorException lex) {
+					context.LogMessage (lex.MessageContainer);
+					if (ex.InnerException != null)
```

Comment:
I'm still don't like this one. Every Exception can have InnerException why to show only this one and why to show only 1 level of depth? If we are printing to console, let's print everything and for all exceptions.

Otherwise LGTM